src/components/views/dialogs/InviteDialog.tsx

```diff
@@ -825,6 +825,10 @@ export default class InviteDialog extends React.PureComponent<IInviteDialogProps
         }
     };
 
+    private onCancel = () => {
+        this.props.onFinished([]);
```

Comment:
If I remember correctly, the first parameter of this `onFinished` callback is a `boolean` to notify whether the modal closed successfully or in an error state

Reply:
This one appears to be special and unique and gets passed the list of strings to invite instead... :/